diff --git a/node_modules/sequelize-auto-migrations/bin/makemigration.js b/node_modules/sequelize-auto-migrations/bin/makemigration.js
index dcbaa49..ca23039 100755
--- a/node_modules/sequelize-auto-migrations/bin/makemigration.js
+++ b/node_modules/sequelize-auto-migrations/bin/makemigration.js
@@ -69,9 +69,9 @@ try {
 } catch (e) { }
 
 //console.log(path.join(migrationsDir, '_current.json'), JSON.parse(fs.readFileSync(path.join(migrationsDir, '_current.json') )))
-let sequelize = require(modelsDir).sequelize;
+let moduleRunners = require(modelsDir);
 
-let models = sequelize.models;
+let {sequelize,...models} = moduleRunners; 
 
 currentState.tables = migrate.reverseModels(sequelize, models);
     
